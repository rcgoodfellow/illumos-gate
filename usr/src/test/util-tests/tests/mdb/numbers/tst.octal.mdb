0o775=K
0o7_7_5__=K
0o7______=K
0t8$d
0123_____456=K
000000___________1=K
