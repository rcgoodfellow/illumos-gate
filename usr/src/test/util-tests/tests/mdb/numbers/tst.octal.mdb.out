                1fd             
                1fd             
                7               
radix = 8 base ten
                a72e            
                1               
